reframe/frontend/cli.py

```diff
@@ -197,14 +197,12 @@ def main():
 
     # Load configuration
     try:
-        settings = config.load_from_file(options.config_file)
+        settings = config.load_settings(options.config_file)
```

Comment:
Let me a bit picky...
Can we stick with the `load_from_file`? Or have it `load_settings_from_file`? Or just `settings_from_file` or `from_file`?
I think it is important to stick to the "from file" part. It makes easier to understand the code

Reply:
I didn't want to use the `load_from_file`, cos this makes more sense as a method of class. Nonetheless, I will change it to `settings_from_file()` or `load_settings_from_file()`.